tests/mocked_plans/sample_plans/9eb9af6a-b919-4cf9-8f2f-9df16a1556be.yaml

```diff
@@ -2,17 +2,13 @@ plan: |
     Upgrade cloud from 'ussuri' to 'victoria'
         Back up MySQL databases
         Archive old database data on nova-cloud-controller
-        OVN subordinate upgrade plan
-            Upgrade plan for 'octavia-ovn-chassis' to 'victoria'
-                WARNING: Changing 'octavia-ovn-chassis' channel from latest/stable to 22.03/stable. This may be a charm downgrade, which is generally not supported.
-                Wait for up to 300s for app 'octavia-ovn-chassis' to reach the idle state
-            Upgrade plan for 'ovn-chassis' to 'victoria'
-                WARNING: Changing 'ovn-chassis' channel from latest/stable to 22.03/stable. This may be a charm downgrade, which is generally not supported.
-                Wait for up to 300s for app 'ovn-chassis' to reach the idle state
         Control Plane subordinate(s) upgrade plan
             Upgrade plan for 'barbican-vault' to 'victoria'
                 Upgrade 'barbican-vault' from 'ussuri/stable' to the new channel: 'victoria/stable'
                 Wait for up to 300s for app 'barbican-vault' to reach the idle state
+            Upgrade plan for 'ceilometer-agent' to 'victoria'
+                Upgrade 'ceilometer-agent' from 'ussuri/stable' to the new channel: 'victoria/stable'
+                Wait for up to 300s for app 'ceilometer-agent' to reach the idle state
```

Comment:
**question:** 

This is a little bit weird for me, according to the definition of data plane. `ceilometer-agent` should belong to data-plane subordinate group. But it's under Control plane subordinates now.

Reply:
There is no data-plane subordinate group now; only subordinate applications. Maybe we could rename the group? What do you think?

Reply:
I remember we discuss this before. The upgrade order should be:

`Control plane subordinate` -> `Control plane` -> `Data plane subordinate` -> `Data plane`

So we need another group.

Reply:
What's wrong with `subordinates` -> `control plane` -> `data plane`? That would simplify things - see internal discussion [here](https://chat.canonical.com/canonical/pl/1t7fugscdfb4pnm9rnc4ie6uzw).

I mean, we could follow your proposed upgrade order, but that makes things more complicated again.

Note that currently the upgrade order in COU is `contral plane subordinate` -> `control plane` -> `nova-compute` -> `data plane principals` -> `data plane subordinate`.

Reply:
> What's wrong with subordinates -> control plane -> data plane?

I think the point is we shouldn't touch anything on data plane before we finish the upgrade on control-plane. The change you need to do is move the whole plane to:

`OVN-chassis` -> `control plane subordinate` -> control plane -> data plane subordinate -> nova-compute -> data plane principals

Reply:
We are kind of upgrading data-plane subordinates first because of [ovn-chassis](https://github.com/canonical/charmed-openstack-upgrader/pull/518). I don't see a strong reason to keep this separation and increase the business logic inside COU.

I think Samuel's changes is just simplifying this to see subordinates as a group.

Reply:
Group heading updated

Reply:
Ok, let's make it easier. Thanks for explanation.